src/containers/AboutIndex/components/WorkDoneCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import Paper from 'components/layout/Paper';
+import React from 'react';
+import MarkdownRenderer from 'components/miscellaneous/MarkdownRenderer';
+import BuildIcon from '@material-ui/icons/Build';
+
+export type WorkDoneCardProps = {
+  changelogURL: string;
+  title: string;
+};
+
+const useStyles = makeStyles((theme) => ({
+  icons: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const WorkDoneCard = ({ title, changelogURL }: WorkDoneCardProps) => {
+  const [markdownString, setMarkDownString] = React.useState<string>('');
+  const markdownHeaderDelimiter = /(?=\n##\s *)/g;
+  const latestVersionIndex = 3;
+  const classes = useStyles();
+
+  React.useEffect(() => {
+    fetch(changelogURL)
+      .then((response) => response.text())
+      .then((text) => {
+        const markdownSections = text.split(markdownHeaderDelimiter);
+        setMarkDownString(markdownSections[latestVersionIndex]);
```

Comment:
Dette kan slås sammen til en linje
```suggestion
        setMarkDownString(text.split(markdownHeaderDelimiter)[latestVersionIndex]);
```

Reply:
Det kan det, men jeg synes det er mer lesbart og enklere å forstå slik det står nå.

Reply:
Den er grei 👍